modules/dynamic-plugins/ref-rh-supported-plugins.adoc

```diff
@@ -112,7 +112,7 @@
 |`./dynamic-plugins/dist/backstage-plugin-techdocs`
 
 
-|TechDocs  |`https://npmjs.com/package/@backstage/plugin-techdocs-backend/v/1.11.1[@backstage/plugin-techdocs-backend]` |1.11.1 
+|TechDocs  |`https://npmjs.com/package/@backstage/plugin-techdocs-backend/v/1.11.5[@backstage/plugin-techdocs-backend]` |1.11.5 
```

Comment:
```suggestion
|TechDocs  |`https://npmjs.com/package/@backstage/plugin-techdocs-backend/v/1.11.5[@backstage/plugin-techdocs-backend]` |1.11.1 
```
[v1.11.1 corresponds to the BS 1.32.6 version](https://github.com/backstage/backstage/blob/v1.32.6/plugins/techdocs-backend/package.json#L3)